components/console-backend-service/internal/domain/k8s/k8s.go

```diff
@@ -46,17 +46,20 @@ func New(restConfig *rest.Config, informerResyncPeriod time.Duration, applicatio
 
 	informerFactory := informers.NewSharedInformerFactory(clientset, informerResyncPeriod)
 
-	namespaceSvc, err := newNamespaceService(informerFactory.Core().V1().Namespaces().Informer(), client)
+	podService := newPodService(informerFactory.Core().V1().Pods().Informer(), client)
+	namespaceSvc, err := newNamespaceService(informerFactory.Core().V1().Namespaces().Informer(), podService, client)
 	if err != nil {
 		return nil, errors.Wrap(err, "while creating namespace service")
 	}
+	namespaceResolver := newNamespaceResolver(namespaceSvc, applicationRetriever, systemNamespaces, podService)
+	namespaceResolver.namespaceConverter = *newNamespaceConverter(systemNamespaces)
```

Comment:
It's already set [here](https://github.com/kyma-project/kyma/pull/5551/files#diff-e6a7d56d82d12a25a102bac0d6b8fea7R42). You probably don't need to set it once again.
```suggestion
```